pkg/cmd/codespace/create.go

```diff
@@ -271,6 +281,10 @@ func (a *App) Create(ctx context.Context, opts createOptions) error {
 		DisplayName:            opts.displayName,
 	}
 
+	if opts.useWeb {
+		return a.browser.Browse(fmt.Sprintf("https://github.com/codespaces/new?repo=%d&ref=%s&machine=%s&location=%s", createParams.RepositoryID, createParams.Branch, createParams.Machine, createParams.Location))
```

Comment:
I could see it either way, I would think that since the user already specified the repo when they ran the command they would have picked the one they wanted, but maybe that's not the case always. It's a minor thing, so I'm find to not use the flag, was just letting you know it was an option.

Reply:
Okay, I think it would be better to leave the repo select visible to maintain flexibility.

But still, if you think it's better to hide it, please let me know and I'll make it happen.